app/components/Distribution.tsx

```diff
@@ -150,18 +154,28 @@ const Distribution = ({ subjectArea, catalogNumber }: DistributionProps) => {
                 legend: {
                   display: false,
                 },
+              tooltip: {
+                callbacks: {
+                  label: (context) => {
+                    const value = context.parsed.y;
+                    const pct = ((Number(value) / totalGradeCountForInstructorNameForTerm) * 100).toFixed(1);
+                    return `${value} (${pct}%)`
+                  }
+                }
+              }
               },
               scales: {
                 y: {
                   ticks: {
+                    count: 7,
                     callback: (value) => {
                       const decimal =
                         Number(value) / totalGradeCountForInstructorNameForTerm;
                       return `${(decimal * 100).toFixed(1)}%`;
                     },
                   },
                   min: 0,
-                  max: totalGradeCountForInstructorNameForTerm,
+                  max: Math.min((15/14) * maxGradeCount, totalGradeCountForInstructorNameForTerm),
```

Comment:
any reason for choosing 15/14 or did it just look nicer with that number? any reason we can't just use `maxGradeCount` by itself?

Reply:
also, if we keep this, would suggest adding a comment along the lines of "if only one distinct letter grade was given, make 100% the max; otherwise leave some space at the top because ... [reason]"

Reply:
Yeah, 15/14 is just stylistically what looked the best -- `maxGradeCount` on its own made the chart look choppy. The current code leaves a top-margin of 1/14th of `maxGradeCount` on the chart. Added the comment.